package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "name": "botbuilder-js",
+  "version": "4.12.0",
```

Comment:
With this change I think it would be prudent to update the shipped package.jsons' versions to `"4.12.0"`/`"4.12.0-preview"` or `"4.0.0-local"`. I would not block this PR on those changes.

Reply:
I actually want to change all our package files to `1.0.0` or even `0.0.0` as we don't do any versioning of packages. The intention is that the _only_ thing manipulating package versions is the `yarn update-versions` script. What do you think?

Reply:
The reason for that is we would now just have a single source of truth for the version across all packages. We would then, eventually, move the DevOps pipeline to omit a version number entirely and just use whatever we have in the repo. Versioning is then properly encoded within the history of the repo.

Reply:
Exactly, the fact that the version number doesn't exist anywhere in SCM is a bad practice to be in.

DevOps and untracked human movements shouldn't be the determining factor in publishing stable/GA bits with a version number.

The only time a DevOps version number might be used is publishing of daily or rc bits.

Reply:
The daily builds only trigger on new commits, so we shouldn't need to add the DevOps Build number to the pipelines. (Also the daily build numbers are unhelpful to anyone by the dev team)

The only way that comes to mind for arriving at a duplicate version number is if someone manually triggered the daily without checking if it exists on npm `@next` after a daily was released that day.

Similarly, we could have `4.12.0-dev.20201110.dec6519` but never `4.12.0-dev.<some-future-date>.dec6519`.